go/private/library.bzl

```diff
@@ -218,7 +226,7 @@ def emit_go_compile_action(ctx, sources, libs, lib_paths, direct_paths, out_obje
     args += ["-src", src]
   for dep in direct_paths:
     args += ["-dep", dep]
-  args += ["--", "-o", out_object.path, "-trimpath", ".", "-I", "."]
+  args += ["-o", out_object.path, "-trimpath", ".", "-I", "."]
```

Comment:
Yeah. I had already spotted that.
I was trying to avoid the -- in favour of compile actually understanding the command line so it can really deal with any args it sees. For instance if someone puts a -I in the gc_goopts it also needs to be abs'd, but for now....